scripts/superdesk/editor2/add-embed.ctrl.js

```diff
@@ -157,15 +157,15 @@ function SdAddEmbedController (embedService, $element, $timeout, $q, _, EMBED_PR
     // toggle when the `extended` directive attribute changes
     $scope.$watch(function() {
         return vm.extended;
-    }, function(extended) {
+    }, function(extended, wasExtended) {
```

Comment:
What is `extended` and `wasExtended`? Some documentation would be nice. It might that I'm just out of context but it's hard for me to figure out what this stuff is.